Plugins/FastJet/include/Acts/Plugins/FastJet/TrackJets.ipp

```diff
@@ -0,0 +1,60 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <Acts/Definitions/Algebra.hpp>
+#include <Acts/Definitions/Common.hpp>
+
+template <typename TrackContainer>
+Acts::FastJet::TrackJetSequence<TrackContainer> Acts::FastJet::makeTrackJets(
+    TrackContainer& tracks, fastjet::JetDefinition jetDef) {
+  std::vector<fastjet::PseudoJet> inputs;
+
+  for (std::size_t i = 0; i < tracks.size(); i++) {
+    typename TrackContainer::ConstTrackProxy track = tracks.getTrack(i);
+    Acts::Vector3 p = track.momentum();
+    float m = track.particleHypothesis().mass();
+
+    float px = p[Acts::eMom0];
+    float py = p[Acts::eMom1];
+    float pz = p[Acts::eMom2];
+    float e = std::sqrt(m * m + px * px + py * py + pz * pz);
```

Comment:
MIght be useful to add a `fourMomentum` getter to the track proxy at some point.

Reply:
Done